roles/terraform-templates/files/terraform-templates-modules-0a1/server/01-input.tf

```diff
@@ -59,6 +54,17 @@ variable experiments {
   type = "string"
 }
 
+variable dev {
```

Comment:
Ah, I guess this is what enables `NOAUTORUN` (regarding my comment above)?

Reply:
yup.